Tools/SystemDebug/sample.json

```diff
@@ -1,9 +1,8 @@
 {
   "guid": "668A3FF9-12C3-4559-8AF4-54B6792122D3",
   "name": "Intel System Debugger Tutorial (Overview)",
-  "categories": ["Toolkit/Intel® oneAPI IoT Toolkit/System Debugger"],
+  "categories": ["Toolkit/Intel® System Bring-Up Toolkit/System Debugger"],
   "description": "Demonstrate basic functionality of the Intel System Debugger.",
-  "os": ["linux", "windows"],
```

Comment:
as this seems to be a power  shell script, the OS  should be windows

Reply:
The powershell script is to flash the efi_application - I was told to remove the OS as it shouldn't show up in the samples browser in visual studio or eclipse

Reply:
Thanks for the clarification